Demo/Common/IntegrationExample.swift

```diff
@@ -86,13 +86,21 @@ internal final class IntegrationExample {
     internal func finish(with error: Error) {
         let isCancelled = ((error as? ComponentError) == .cancelled)
 
-        currentComponent?.stopLoading(withSuccess: false) { [weak self] in
+        stopLoadingIfNeeded(false) { [weak self] in
             self?.presenter?.dismiss { [weak self] in
                 if !isCancelled { self?.presentAlert(with: error) }
             }
         }
     }
 
+    private func stopLoadingIfNeeded(_ success: Bool, completion: (() -> Void)?) {
```

Comment:
I have a feeling that `stopLoadingIfNeeded` is redundant: 
the `success: Bool` is never used, so it is basically a `Cancelable` with `didCancel(completion: (() -> Void)?)`

Reply:
`success: Bool` is used, not sure what you mean? not sure even what you mean by "redundant"? cancellable is when the user cancel the component, this is after the payment is concluded by success or failure.